### DISCUSSION TITLE
RHVoice API

### DISCUSSION THREAD

**Discussion by igoryonya** [RHVoice/RHVoice#388] [Q&A]
Здравствуйте, есть API для RHVoice?
Я хочу написать GUI на Perl, использующее RHVoice движок сначала, т.к. не нашёл ни одного GUI для RHVoice и ни одного нормального GUI для TTS вообще на Linux.

**Comment by alex19EP** [comment]
api находится 
[тут](https://github.com/RHVoice/RHVoice/tree/master/src/include)

**Reply by igoryonya** [reply]
Значит в коде смотреть, Документации нет. Правильно понял?

**Reply by alex19EP** [reply]
да к сожалению документацию мы пока не сделали.

**Reply by igoryonya** [reply]
Oh, so complex. It will take me a lot of time to figure it out, so I think, I will just do the plane old binary system calls for now.
